Integrate prometheus. Allow custom metrics (#12)
mia-platform/lc39: docs/main-entrypoint.md

Both of this endpoints are set to permanently run on log level `silent` for decreasing the amount of noise in the
logs during the deployment.

## Prometheus Metrics

> By default `lc39` exposes `/-/metrics` endpoint for prometheus scraper. In this metrics there're some process, garbage collection and http information.

Reviewer: ```suggestion
By default `lc39` exposes `/-/metrics` endpoint for prometheus scraper. In this metrics there are some process, garbage collection and http information.
```

> By default `lc39` exposes `/-/metrics` endpoint for prometheus scraper. In this metrics there're some process, garbage collection and http information.

Reviewer 2: What do you mean by "In this metrics there're some process, garbage collection and http information." ? Is it the purpose of the endpoint? Can we write it more clearly?

Something on the line of:

> The endpoint is responsible of exposing relevant information about running processes, http request information (current load, errors, etc)

What do you think?

Reviewer 2: Is it only an empty route or does it have some information provided by default? Can we clarify this?

Also is there some doc about the shape of the object that must be returned by the endpoint?

Author: `The response body contains process, garbage collection and http information.`
?

> By default `lc39` exposes `/-/metrics` endpoint for prometheus scraper. In this metrics there are some process, garbage collection and http information.

Author: ```suggestion
By default `lc39` exposes `/-/metrics` endpoint for [Prometheus](https://prometheus.io/). The response body contains process, garbage collection and http information
```
